src/node_buffer.cc

```diff
@@ -197,7 +298,7 @@ Local<Object> New(Environment* env,
 }
 
 
-Local<Object> Use(Isolate* isolate, char* data, uint32_t length) {
+Local<Object> Use(Isolate* isolate, char* data, size_t length) {
```

Comment:
If the function prototype is going to change anyway, leave it in.  `size_t` is the more appropriate type for number-of-elements variables.